app/uk/gov/hmrc/servicedependencies/controller/admin/AdministrationController.scala

```diff
@@ -28,9 +28,9 @@ import uk.gov.hmrc.servicedependencies.service.DependencyDataUpdatingService
 class AdministrationController @Inject()(dependencyDataUpdatingService: DependencyDataUpdatingService)
     extends BaseController {
 
-  def reloadLibraryDependenciesForAllRepositories() = Action { implicit request =>
+  def reloadLibraryDependenciesForAllRepositories(force: Option[String] = None) = Action { implicit request =>
```

Comment:
maybe `Option[Boolean]` instead?

Reply:
I tried that in the first case, but since this is used directly in a route, it seems that we're forced to use `String` or `Option[String]`.

Reply:
Are we? There is this concept of PathBindable, but maybe it doesn't apply to query string params

Reply:
see this: https://www.playframework.com/documentation/2.5.x/ScalaRequestBinders#querystringbindable